public/src/components/channelManagement/testVariantsSplitEditor.tsx

```diff
@@ -169,7 +166,7 @@ const TestVariantsSplitEditor: React.FC<TestVariantsSplitEditorProps> = ({
           />
         </RadioGroup>
 
-        {controlProportionSettings && hasControl(variants) && (
+        {controlProportionSettings && (
```

Comment:
I've moved the `hasControl` check outside this component so we don't render the whole component if we don't have a control